### ISSUE TITLE
Question  on XNNPACK support for PowerPC

### ISSUE THREAD

**Issue by vinithakv** [pytorch/pytorch#115820]
Hi,
 We are starting to work on XNNPACK support for PowerPC. Aa an initial step the support for PowerPC in XNNPACK is enabled with scalar kernels and is submitted to XNNPACK repo. Later we want to integrate this XNNPACK with PyTorch.   
I have a question regarding the PyTorch and XNNPACK. 
The XNNPACK version in PyTorch uses is a bit old. Do you have a plan to  upgrade the XNNPACK  ? 

Regards,
Vinitha

**Comment by mikaylagawarecki** [comment]
cc @malfet not sure whether you are the right person to tag here regarding third_party library versioning

**Comment by malfet** [comment]
Yes, there are plans to update XNNPACK, see https://github.com/pytorch/pytorch/pull/115587 (which got merged but then reverted)

**Comment by mcr229** [comment]
I believe XNNPACK has been updated in this PR:

https://github.com/pytorch/pytorch/pull/115714

The commit version PyTorch uses is now here: https://github.com/google/XNNPACK/commit/d9cce341f86a207da9d851d05e26cd50b508b73c
